tests/test-values.yaml

```diff
@@ -0,0 +1,220 @@
+      - container
+      - container_name
+      - component
+      - workload
+      - job
+      - scrape_job
+      ingestion_burst_size_mb: 100
+      ingestion_rate_mb: 50
+      max_global_streams_per_user: 10000
+      max_line_size: 10MB
+      retention_period: 744h
+      shard_streams:
+        enabled: true
+      split_queries_by_interval: 12h
+      volume_enabled: true
+    pattern_ingester:
+      enabled: true
+    rulerConfig:
+      remote_write:
+        add_org_id_header: false
+        clients:
+          mimir:
+            url: http://mimir-gateway.mimir/api/v1/push
+        enabled: true
+      wal:
+        dir: /var/loki/ruler-wal
+    schemaConfig:
+      configs:
+      - from: "2023-01-24"
+        index:
+          period: 24h
+          prefix: loki_index_
+        object_store: s3
+        schema: v12
+        store: boltdb-shipper
+      - from: "2024-01-02"
+        index:
+          period: 24h
+          prefix: loki_index_
+        object_store: s3
+        schema: v12
+        store: tsdb
+      - from: "2024-05-23"
+        index:
+          period: 24h
+          prefix: loki_index_
+        object_store: s3
+        schema: v13
+        store: tsdb
+    server:
+      grpc_server_max_recv_msg_size: 104857600
+      grpc_server_max_send_msg_size: 104857600
+    tracing:
+      enabled: "false"
+  monitoring:
+    serviceMonitor:
+      enabled: true
+  rbac:
+    pspEnabled: false
+  read:
+    extraArgs:
+    - -config.expand-env
+    - -querier.multi-tenant-queries-enabled
+  resultsCache:
+    enabled: false
+  serviceAccount:
+    annotations:
+      eks.amazonaws.com/role-arn: arn:aws:iam::242036376510:role/giantswarm-golem-loki
+  sidecar:
+    image:
+      repository: gsoci.azurecr.io/giantswarm/k8s-sidecar
+  write:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: eks.amazonaws.com/compute-type
+              operator: NotIn
+              values:
+              - fargate
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            labelSelector:
+              matchLabels:
+                app.kubernetes.io/component: write
+                app.kubernetes.io/instance: loki
+                app.kubernetes.io/name: loki
+            topologyKey: kubernetes.io/hostname
+          weight: 1
+        requiredDuringSchedulingIgnoredDuringExecution: []
+    autoscaling:
+      behavior:
+        scaleUp:
+          policies:
+          - periodSeconds: 120
+            type: Pods
+            value: 1
+      maxReplicas: 25
+    extraArgs:
+    - -config.expand-env
+    persistence:
+      size: 40Gi
+    resources:
+      limits:
+        memory: 2Gi
+      requests:
+        cpu: 100m
+        memory: 1Gi
+      policyTypes:
+        - Egress
+multiTenantAuth:
+  deployCredentials: true
```

Comment:
I would test without mtp

Reply:
But really nice work

Reply:
Why that ? :thinking:

Reply:
Well you're not testing that it's ready right?

Reply:
Well yeah, you have a point there.